sdk/cosmos/azure-cosmos/azure/cosmos/_routing/aio/routing_map_provider.py

```diff
@@ -54,7 +54,7 @@ async def get_overlapping_ranges(self, collection_link, partition_key_ranges):
         overlapping partition key ranges.
 
         :param str collection_link: The name of the collection.
-        :param list partition_key_range: List of partition key range.
+        :param list partition_key_ranges: List of partition key range.
```

Comment:
This looks like it was public API?

Reply:
the routing module is already supposed to be private, so this method does not/ should not get used right?

Reply:
I think before we go GA we should make most of these files in `aio` directory private. It was a big mistake to make them public in the sync version.... and I don't think we gain anything by repeating it here....

Reply:
I'll add this to the list of issues to take care of before GA then, thanks Anna!